jobs/JEVS_MESOSCALE_STATS

```diff
@@ -88,11 +88,11 @@ if [ $MODELNAME = nam ] || [ $MODELNAME = rap ]; then
 elif [ $MODELNAME = sref ]; then
   export VDATE=${VDATE:-$PDYm2}
 
-  export COMIN=${COMIN:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT/$RUN
+  export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}/prep/$COMPONENT/$RUN
   export COMINsref=${COMINsref:-$(compath.py  sref/$sref_ver)}
   export COMINprepbufr=${COMINprepbufr:-$(compath.py  obsproc/${obsproc_ver})}
-  export COMINccpa=${COMINccpa:-$(compath.py  ccpa/${ccpa_ver})}
-  export COMINsrefmean=${COMINsrefmean:-$(compath.py  $NET/$evs_ver)}/prep/$COMPONENT
+  export EVSINccpa=${EVSINccpa:-$(compath.py  ccpa/${ccpa_ver})}
```

Comment:
Is this supposed to be EVS data or  model data? If it is model data, then the variable should be
```
export COMINccpa=${COMINccpa:-$(compath.py $envir/com/ccpa/${ccpa_ver})}
```

Reply:
Changing to COMINccpa.

Reply:
> Changing to COMINccpa for stats and leaving to EVSINccpa for plots, because the plot scripts use them from EVS prep folder

Ok, that makes sense, @RoshanShrestha-NOAA